test/views/components/GovukFooterSpec.scala

```diff
@@ -40,8 +40,7 @@ class GovukFooterSpec extends ViewSpec with ViewFns with GuiceOneAppPerSuite {
     val footerLinks = app.injector.instanceOf[FooterLinks]
     val footer = app.injector.instanceOf[views.html.components.govukFooter]
     implicit def messages: Messages = messagesApi.preferred(Seq(Lang("en")))
-    val fakeRequest = FakeRequest("GET", "/")
-
+   implicit val fakeRequest = FakeRequest("GET", "/")
```

Comment:
sorry, OCD kicking in, can you align up line 43 :-) 
Also, when there, can you tidy up imports as quite a few not used (not because of this change)

Reply:
Removed